radiation/python/radsw/swrad_gt4py.py

```diff
@@ -0,0 +1,1751 @@
+import numpy as np
+from numpy.lib.shape_base import _column_stack_dispatcher
+import xarray as xr
+import sys
+import time
+
+sys.path.insert(0, "..")
+from util import compare_data, create_storage_from_array, create_storage_zeros
+from config import *
+from stencils_sw_gt4py import *
+from radsw.radsw_param import idxebc, nspa, nspb, ngs, ng, NGB, idxsfc
+from radphysparam import iswrate
+from phys_const import con_g, con_cp
+
+sys.path.append(SERIALBOX_DIR + "/python")
+import serialbox as ser
+
+rebuild = False
+validate = True
+backend = "gtc:gt:cpu_ifirst"
+
+# Flag to say whether or not to do intermediate tests
+do_test = False
+
+ddir = "../../fortran/data/SW"
+ddir2 = "../../fortran/radsw/dump"
+
+serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, "Generator_rank1")
+serializer2 = ser.Serializer(ser.OpenModeKind.Read, ddir2, "Serialized_rank1")
```

Comment:
why are these in different directories? is this data from the full fortran model vs the standalone?

Reply:
Yes, `fortran/data/SW` is the data from fv3gfs-fortran, and the `dump` directory is for the standalone